src/main/scala/nl.knaw.dans.easy.managedeposit/DepositPropertiesTable.scala

```diff
@@ -0,0 +1,356 @@
+         |  number_of_continued_deposits
+         |FROM deposit_properties
+         |${
+        if (whereConditions.isEmpty) ""
+        else "WHERE " + whereConditions.mkString(" AND\n")
+      }
+         |ORDER BY last_modified
+         |""".stripMargin
+
+    debug(s"query = $query")
+
+    managed(c.prepareStatement(query))
+      .map(ps => {
+        val resultSet = ps.executeQuery()
+        while (resultSet.next()) {
+          val propsText = resultSet.getString("properties")
+          val props = new PropertiesConfiguration() {
+            setDelimiterParsingDisabled(true)
+            load(new StringReader(propsText))
+          }
+
+          action(
+            DepositInformation(
+              depositId = resultSet.getString("uuid"),
+              doiIdentifier = props.getString("identifier.doi", "n/a"),
+              dansDoiRegistered = Option(BooleanUtils.toBoolean(props.getString("identifier.dans-doi.registered"))),
+              fedoraIdentifier = props.getString("identifier.fedora", "n/a"),
+              depositor = resultSet.getString("depositor_user_id"),
+              state = State.toState(resultSet.getString("state_label")).getOrElse(State.UNKNOWN),
+              description = props.getString("state.description", "n/a"),
+              creationTimestamp = props.getString("creation.timestamp", "n/a"),
+              numberOfContinuedDeposits = resultSet.getInt("number_of_continued_deposits"),
+              storageSpace = resultSet.getLong("storage_size_in_bytes"),
+              lastModified = new DateTime(resultSet.getTimestamp("last_modified")).withZone(DateTimeZone.UTC).toString(dateTimeFormatter),
+              origin = props.getString("deposit.origin", "n/a"),
+              location = resultSet.getString("location"),
+              bagDirName = props.getString("bag-store.bag-name", "n/a"),
+              datamanager = resultSet.getString("datamanager"),
+              hasBag = resultSet.getBoolean("has_bag")
+            ))
+        }
+      }).tried.doIfFailure {
+      case NonFatal(e) => logger.error("Error while processing deposit information", e)
+    }
+  }
+
+  def getDepositSizeAndSpaceGroupedByState(depositor: Option[DepositorId], datamanager: Option[Datamanager], ageInDays: Option[Age]): Try[Seq[(State, Long, Long)]] = {
+    database.doTransaction(implicit c => selectStateSizeAndSpace(depositor, datamanager, ageInDays))
+  }
+
+  private def selectStateSizeAndSpace(depositor: Option[DepositorId], datamanager: Option[Datamanager], ageInDays: Option[Age])(implicit c: Connection): Try[Seq[(State, Long, Long)]] = {
+    trace(())
+    val whereConditions = new ListBuffer[String]()
+    if (depositor.nonEmpty) whereConditions.append(s"depositor_user_id = '${ depositor.get }'")
+    if (datamanager.nonEmpty) whereConditions.append(s"datamanager = '${ datamanager.get }'")
+    if (ageInDays.nonEmpty) whereConditions.append(s"last_modified > NOW() - INTERVAL '${ ageInDays.get } days'")
```

Comment:
More functional would be something like
```suggestion
    val whereConditions = Seq(
      depositor.map(d -> s"depositor_user_id = '$d'"),
      datamanager.map(m -> s"datamanager = '$m'"),
      ageInDays.map(i -> s"last_modified > NOW() - INTERVAL '$i days'"),
    )
```
Also move the if-else of line 162 to here.